reference/cd-aws-code-services.md

```diff
@@ -0,0 +1,91 @@
+---
+title: AWS Code Services
+---
+
+[Amazon Code Services](https://aws.amazon.com/products/developer-tools/) encompases a variety
+of specific tools for CI/CD, including [CodePipeline](https://aws.amazon.com/codepipeline/),
+[CodeBuild](https://aws.amazon.com/codebuild/), [CodeDeploy](https://aws.amazon.com/codedeploy/),
+and others.
+
+To incorporate updating Pulumi stacks into an AWS Code Services-managed CI/CD system, you'll
+want to use CodeBuild. Pulumi needs to execute a built program in order to determine the desired
+state of cloud resources, and CodeBuild provides a compute environment to do just that.
+
+If you are using CodePipeline, you can then create a new pipeline stage which triggers the
```

Comment:
It would be cool eventually to have a Pulumi program for this!

Reply:
Added the idea to the Hackathon Ideas wiki. This would come in handy in a lot of scenarios when using AWS.